model/config.go

```diff
@@ -1490,6 +1498,26 @@ func (o *Config) SetDefaults() {
 		o.PluginSettings.Plugins = make(map[string]interface{})
 	}
 
+	if o.MessageExportSettings.EnableExport == nil {
+		o.MessageExportSettings.EnableExport = new(bool)
+		*o.MessageExportSettings.EnableExport = false
```

Comment:
You could change these to the `o.Group.Setting = NewBool(false)` syntax using the new utility functions if you want.

Reply:
On the way in an upcoming commit. Those functions are newer than this code.